include/uapi/linux/virtio_ids.h

```diff
@@ -48,5 +48,6 @@
 #define VIRTIO_ID_FS           26 /* virtio filesystem */
 #define VIRTIO_ID_PMEM         27 /* virtio pmem */
 #define VIRTIO_ID_MAC80211_HWSIM 29 /* virtio mac80211-hwsim */
+#define VIRTIO_ID_WATCHDOG     35 /* (temporary) virtio-watchdog */
```

Comment:
Are you anticipating that it'll take time to get this accepted, so better to take a number higher than 30?

Reply:
This was the next available number in the spec repo.